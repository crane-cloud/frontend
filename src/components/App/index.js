import React from "react";
// import LandingPage from "../../pages/LandingPage";
import "./App.css";
import LoginPage from "../../pages/LoginPage";

const App = () => (
  <div className="LandingPage">
    <LoginPage />
    {/* <LandingPage /> */}
  </div>
);

export default App;
