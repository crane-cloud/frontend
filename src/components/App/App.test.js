import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './index';

describe('App Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<App />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
