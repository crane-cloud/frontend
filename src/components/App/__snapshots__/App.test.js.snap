// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match the snapshot 1`] = `
<div
  className="LandingPage"
>
  <Connect(withRouter(LandingPage)) />
</div>
`;
